app/frontend/components/EnrollmentForm.jsx

```diff
@@ -7,8 +7,6 @@ import { TimePicker } from '@mui/x-date-pickers/TimePicker';
 import { DesktopDatePicker } from '@mui/x-date-pickers/DesktopDatePicker';
 import "bootstrap/dist/css/bootstrap.min.css";
 import { Button, Form, Card, Row } from "react-bootstrap";
-// import { MobileDatePicker } from '@mui/x-date-pickers/MobileDatePicker';
```

Comment:
Did not mean to delete this comment. We can add it back in if wanted.

Reply:
I think this should be ok. Since it's not expected of the date time picker to be used with a smartphone.